### PR TITLE
Update go-velocity and add go-pos and go-pos-unsafe test

### PR DESCRIPTION
**Pull request by knorth55**
I update `:go-velocity` test to work properly.
I also add `test-go-pos` and `test-go-pos-unsafe`.

this PR includes https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/411 and https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/412
`test-go-pos-unsafe` is for https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/406.

### CODE REVIEW

File: pr2eus/pr2-interface.l [hunk 1]
```diff
@@ -90,7 +90,7 @@
      (:worldcoords
       (unless joint-action-enable
 	(return-from :state (send self :worldcoords)))
-      (send-super :state :worldcoords (or (cadr args) "world")))
+      (send-super :state :worldcoords (or (cadr args) "map")))
```
Comment (k-okada) [inline pr2eus/pr2-interface.l]: @knorth55 why we need to change this. Seems very dangerous.

Comment (knorth55) [reply pr2eus/pr2-interface.l]: I remove this modification.

**Comment by knorth55** [comment]
I add test for `:go-pos` and `:go-pos-unsafe`
`:go-pos-unsafe` is override in `test` file, and I could not write it elegantly.

**Comment by knorth55** [comment]
Build failure results:
Indigo:
```
Traceback (most recent call last):

  File "/opt/ros/indigo/share/rostest/cmake/../../../bin/rostest", line 36, in <module>

    rostestmain()

  File "/opt/ros/indigo/lib/python2.7/dist-packages/rostest/__init__.py", line 274, in rostestmain

    _main()

  File "/opt/ros/indigo/lib/python2.7/dist-packages/rostest/rostest_main.py", line 159, in rostestmain

    testCase = rostest.runner.createUnitTest(pkg, test_file, options.reuse_master, options.clear, options.results_base_dir)

  File "/opt/ros/indigo/lib/python2.7/dist-packages/rostest/runner.py", line 229, in createUnitTest

    config = roslaunch.parent.load_config_default([test_file], None)

  File "/opt/ros/indigo/lib/python2.7/dist-packages/roslaunch/config.py", line 455, in load_config_default

    raise RLException(e)

roslaunch.core.RLException: unused args [KINECT1, KINECT2] for include of [/opt/ros/indigo/share/pr2_gazebo/launch/pr2.launch]
```
https://github.com/PR2/pr2_simulator/commit/cda2e39c34072237884c69fdcdac01c8eac34103#diff-41e5fda4ba4f344641c06c6292fa2450
This commits is in `2.0.8` and last release of indigo is `2.0.7`.

Kinetic: Timeout
Melodic: 
```
/usr/bin/make -f CMakeFiles/_run_tests_pr2eus_rostest_test_pr2-ri-test.launch.dir/build.make CMakeFiles/_run_tests_pr2eus_rostest_test_pr2-ri-test.launch.dir/depend

xacro: in-order processing became default in ROS Melodic. You can drop the option.

Traceback (most recent call last):

  File "/opt/ros/melodic/share/rostest/cmake/../../../bin/rostest", line 36, in <module>

    rostestmain()

  File "/opt/ros/melodic/lib/python2.7/dist-packages/rostest/__init__.py", line 274, in rostestmain

    _main()

  File "/opt/ros/melodic/lib/python2.7/dist-packages/rostest/rostest_main.py", line 159, in rostestmain

    testCase = rostest.runner.createUnitTest(pkg, test_file, options.reuse_master, options.clear, options.results_base_dir)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/rostest/runner.py", line 229, in createUnitTest

    config = roslaunch.parent.load_config_default([test_file], None)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/config.py", line 457, in load_config_default

    loader.load(f, config, argv=args, verbose=verbose)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 751, in load

    self._load_launch(launch, ros_config, is_core=core, filename=filename, argv=argv, verbose=verbose)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 723, in _load_launch

    self._recurse_load(ros_config, launch.childNodes, self.root_context, None, is_core, verbose)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 687, in _recurse_load

    val = self._include_tag(tag, context, ros_config, default_machine, is_core, verbose)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 95, in call

    return f(*args, **kwds)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 591, in _include_tag

    inc_filename = self.resolve_args(tag.attributes['file'].value, context)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 185, in resolve_args

    return substitution_args.resolve_args(args, context=context.resolve_dict, resolve_anon=self.resolve_anon)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 392, in resolve_args

    resolved = _resolve_args(resolved, context, resolve_anon, commands)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 405, in _resolve_args

    resolved = commands[command](resolved, a, args, context)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 166, in _find

    source_path_to_packages=source_path_to_packages)

  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 212, in _find_executable

    full_path = _get_executable_path(rp.get_path(args[0]), path)

  File "/usr/lib/python2.7/dist-packages/rospkg/rospack.py", line 203, in get_path

    raise ResourceNotFound(name, ros_paths=self._ros_paths)

rospkg.common.ResourceNotFound: pr2_navigation_slam

ROS path [0]=/opt/ros/melodic/share/ros

ROS path [1]=/home/travis/ros/ws_jsk_pr2eus/src/jsk_pr2eus/pr2eus

ROS path [2]=/home/travis/ros/ws_jsk_pr2eus/src/jsk_pr2eus/pr2eus

ROS path [3]=/home/travis/ros/ws_jsk_pr2eus/src/jsk_pr2eus/pr2eus_moveit

ROS path [4]=/home/travis/ros/ws_jsk_pr2eus/src/jsk_pr2eus/pr2eus_tutorials

ROS path [5]=/opt/ros/melodic/share

Cannot find results, writing failure results to '/home/travis/ros/ws_jsk_pr2eus/build/pr2eus/test_results/pr2eus/MISSING-rostest-test_pr2-ri-test.xml'
```

**Comment by knorth55** [comment]
I fixed the indigo problem by setting env.

**Pull request closed without merging**